src/Models/Link.php

```diff
@@ -68,6 +69,7 @@ public function getCMSFields(): FieldList
 
             $titleField = $fields->dataFieldByName('Title');
             $titleField->setTitle(_t('LinkField.LINK_FIELD_TITLE', 'Title'));
+            $titleField->setDescription(_t('LinkField.LINK_FIELD_TITLE_DESCRIPTION', 'If left blank, default title will be used'));
```

Comment:
```suggestion
            $titleField->setDescription(_t(static::class . '.LINK_FIELD_TITLE_DESCRIPTION', 'If left blank, default title will be used'));
```
Sorry, should have caught this last time. 

The rationale for this is explained in https://github.com/silverstripe/silverstripe-linkfield/pull/126#discussion_r1413228980

Reply:
Updated